src/chainparams.cpp

```diff
@@ -150,6 +164,17 @@ class CTestNetParams : public CMainParams {
         nMinerThreads = 0;
         nPruneAfterHeight = 1000;
 
+        // 1MB max blocks before 1 Aug 2015
+        // Then, if miner consensus: 8MB max, doubling every two years
+        nMaxSizePreFork = 1000*1000; // 1MB max pre-fork
+        nEarliestSizeForkTime = 1438387200; // 1 Aug 2015 00:00:00 UTC
+        nSizeForkActivationTime = std::numeric_limits<int32_t>::max(); // Until miner vote passes, "never"
+        nSizeDoubleEpoch = 60*60*24*365*2; // two years
+        nMaxSizeBase = 8*1000*1000; // 8MB
+        nMaxSizeDoublings = 10;
+        nActivateSizeForkMajority = 75; // 75 of 100 to activate fork
```

Comment:
I agree. (Although, with my merge-mine proposal below, this becomes less relevant. 50% might be sufficient to start the merge-mined fork).